libraries/ashrae/guideline36/4.10-chilled-water-plant.ttl

```diff
@@ -0,0 +1,1221 @@
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Heating_Enable_Command ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    # applications
+    sh:property [
+        rdfs:label "Piping heat trace status" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Enable_Status ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1
+    ] .
+
+:variable-speed-fan-motor a sh:NodeShape, owl:Class, bmotif:HVAC ;
+    rdfs:label "Variable Speed Fan Motor" ;
+    sh:class brick:Variable_Frequency_Drive ;
+    # required
+    sh:property [
+        rdfs:label "Fan Start" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:On_Off_Command ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    # required
+    sh:property [
+        rdfs:label "Fan Status" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:On_Off_Status ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    # required
+    sh:property [
+        rdfs:label "Fan Speed" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Frequency_Command ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    # optional
+    sh:property [
+        rdfs:label "Fan Alarm" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Fault_Status ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1
+    ] .
+
+:variable-speed-fan a sh:NodeShape, owl:Class, bmotif:HVAC ;
+    rdfs:label "Variable Speed Fan" ;
+    sh:class brick:Cooling_Tower_Fan ;
+    sh:property [
+        sh:path brick:hasPart ;
+        sh:qualifiedValueShape [ sh:node :variable-speed-fan-motor ] ;
+        sh:qualifiedMinCount 1
+    ] .
+
+:ct-inlet-isolation-valve a sh:NodeShape, owl:Class, bmotif:HVAC ;
+    rdfs:label "Cooling Tower Inlet Isolation Valve" ;
+    sh:class brick:Isolation_Valve ;
+    # applications
+    sh:property [
+        rdfs:label "CT inlet isolation valve" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Valve_Position_Command ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    # Retain the following two points for optional valve end switch status feedback. Delete otherwise.
+    # optional
+    sh:property [
+        rdfs:label "CT inlet isolation valve closed end switch" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:On_Status ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    # optional
+    sh:property [
+        rdfs:label "CT inlet isolation valve open end switch" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Off_Status ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1
+    ] .
```

Comment:
@selamHaile ⬆️

Reply:
Usually we use open/Close enumeration for the Valve and damper, and on/off is for the Fan and pumps. And the Brick https://brickschema.org/ontology/1.3/classes/Open_Close_Status will show the current valve status (open/closed)

Reply:
Ok. The G36 points are `status` points, with a note that reads...
>Retain the following two points for optional valve end switch status feedback.

Keeping as is since these are two distinct points.